sdk/resourcemanager/azure-resourcemanager-appservice/README.md

```diff
@@ -41,14 +41,14 @@ In addition, Azure subscription ID can be configured via environment variable `A
 
 With above configuration, `azure` client can be authenticated by following code:
 
-<!-- embedme ./src/samples/java/com/azure/resourcemanager/appservice/ReadmeSamples.java#L21-L26 -->
-```java
+```java readme-sample-authenticate
 AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
 TokenCredential credential = new DefaultAzureCredentialBuilder()
     .authorityHost(profile.getEnvironment().getActiveDirectoryEndpoint())
     .build();
-AppServiceManager manager = AppServiceManager
-    .authenticate(credential, profile);
+AzureResourceManager azure = AzureResourceManager
+    .authenticate(credential, profile)
+    .withDefaultSubscription();
```

Comment:
It could be a issue of picking the wrong sample tag. It should use tag from
resourcemanager-appservice/src/samples/java/com/azure/resourcemanager/appservice/ReadmeSamples.java

I could fix it after your PR.

Reply:
I've fixed this, turns out a README root and glob was updating codesnippets in other packages.